EPM Integration Training (#3029)
SkylineCommunications/dataminer-docs: user-guide/Advanced_Modules/EPM/EPM_Integration_Training/Connectors/ConnectorsVisChainsFields.md

    <Field name="Device" options="displayInFilter;ignoreEmptyFilterValues;tabs:2500-KPI;details:2500;ShowBubbleupAndInstanceAlarmLevel" pid="2501">
        <Display>
            <Selection>
                <Visibility default="true">
                    <Standalone pid="310">
                        <Value>1</Value>
                    </Standalone>
                </Visibility>
            </Selection>
        </Display>
    </Field>
</Chain>
```

### Chain configuration

| Tag            | Attribute | Description                                                                                                                                       |
|----------------|-----------|---------------------------------------------------------------------------------------------------------------------------------------------------|
| `<Visibility>` | `default` | Specifies if the chain or field should be displayed by default (`true` or `false`).                                                               |
| `<Standalone>` | `pid`     | Specifies the ID of the configurable parameter linked to toggle visibility.                                                                       |

> | `<Value>`      |           | Defines one of the possible values the parameter must have to toggle the visibility to the opposite setting in relation to the default attribute. |

Reviewer: Could you clarify this? Is this what you mean?

```suggestion
| `<Value>`      |           | Defines the parameter value that will set the visibility to be the opposite of what is defined by default. |
```

Reviewer 2: Yes that is what I ment
